contracts/common/RevisionRevert.sol

```diff
@@ -0,0 +1,64 @@
+pragma solidity 0.8.28;
+pragma abicoder v2;
+
+import {ICallSolana} from "../external/neon-contracts/contracts/precompiles/ICallSolana.sol";
+
+contract RevisionRevert {
```

Comment:
why do you call it as revert? there are no reverts 

all tests for revisions check that revision is changing and this one is the same